specification/storsimple8000series/resource-manager/readme.md

```diff
@@ -56,3 +56,23 @@ csharp:
   output-folder: $(csharp-sdks-folder)/StorSimple8000Series/Management.StorSimple8000Series/Generated
   clear-output-folder: true
 ```
+
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: storsimple

```

Comment:
Today we are generating as `storsimple8000series`

Reply:
This way it matches how it is declared in Swagger. I'd like to keep that 1 to 1 relationship as much as possible.